polaris/ocean/config/single_layer.yaml

```diff
@@ -0,0 +1,19 @@
+omega:
+  time_integration:
+    config_time_integrator: RK4
+  cvmix:
+    config_use_cvmix: false
+  ALE_vertical_grid:
+    config_vert_coord_movement: impermeable_interfaces
+    config_ALE_thickness_proportionality: weights_only
+  debug:
+    config_disable_thick_vadv: true
+    config_disable_thick_sflux: true
+    config_disable_vel_vmix: true
+    config_disable_vel_vadv: true
+    config_disable_tr_all_tend: true
+    config_disable_vel_hmix: true
```

Comment:
I don't think hmix should be off by default for a single layer case

Reply:
That seems right to me. I think it would best to run a single layer case with all hmix options to be sure though.